tests/Drupal/InfoFiles/drupal8/test.info.yml

```diff
@@ -4,4 +4,4 @@ name: Test
 # These should not be here, will be added by drupal.org packaging.
 project: test
 datestamp: 1234567
-version: 1.2
+version: 1.2
```

Comment:
This file should not be modified, please revert the change here.

Reply:
@klausi I'd like to know why we cant use the exitsing `tests/Drupal/InfoFiles/drupal8/test.info.yml` and add the faults to it. The existing tests will all still pass on the changed file. I know sometimes that changing an existing test file causes problems in other tests, but it does not do so here. Or do you like having new test files for every sniff? The .info.yml tests are a bit different from the others. Just want to know our policy, for future enhancements. Thanks

Reply:
I did not see your comment above. OK maybe it is better to have separate test files.

Reply:
@branjansdl1 can you address this and not modify this file?